packages/cojson/src/coValueCore.ts

```diff
@@ -121,14 +121,35 @@ export class CoValueCore {
     this.header = header;
     this._sessionLogs = internalInitSessions;
     this.node = node;
+  }
+
+  groupInvalidationSubscription?: () => void;
+
+  subscribeToGroupInvalidation() {
+    if (this.groupInvalidationSubscription) {
+      return;
+    }
+
+    const header = this.header;
 
     if (header.ruleset.type == "ownedByGroup") {
-      this.node
-        .expectCoValueLoaded(header.ruleset.group)
-        .subscribe((_groupUpdate) => {
-          this._cachedContent = undefined;
-          this.notifyUpdate("immediate");
+      const groupId = header.ruleset.group;
+      const entry = this.node.coValuesStore.get(groupId);
+
+      if (entry.state.type === "available") {
+        this.groupInvalidationSubscription = entry.state.coValue.subscribe(
+          (_groupUpdate) => {
+            this._cachedContent = undefined;
+            this.notifyUpdate("immediate");
+          },
+          false,
+        );
+      } else {
+        logger.error("CoValueCore: Owner group not available", {
+          id: this.id,
+          groupId,
         });
+      }
```

Comment:
For extra safety, I've switched an error log instead of throwing an error.

The group here is required to invalidate the content when the permission change, so the group availability shouldn't be a strong requirement.